R/transform_filter-methods.R

```diff
@@ -183,7 +183,8 @@ rarefy_even_depth <- function(physeq, sample.size=min(sample_sums(physeq)),
     rmtaxa = taxa_names(newsub)[taxa_sums(newsub) <= 0]
     if( length(rmtaxa) > 0 ){
       if(verbose){
-        message(length(rmtaxa), "OTUs were removed because they are no longer \n",
+        message(length(rmtaxa),
+            " OTUs were removed because they are no longer \n",
```

Comment:
Related PR https://github.com/joey711/phyloseq/pull/875

I can rebase my PR if this PR gets merged.